cobbler.spec

```diff
@@ -497,6 +517,11 @@ sed -i -e "s/SECRET_KEY = ''/SECRET_KEY = \'$RAND_SECRET\'/" %{_datadir}/cobbler
 %attr(-,%{apache_user},%{apache_group}) %{apache_dir}/cobbler_webui_content/
 %endif
 
+%if 0%{?suse_version}
+%files tests
+%dir /usr/share/cobbler/tests
+/usr/share/cobbler/tests/*
+%endif
```

Comment:
Use proper macros and drop the `%suse_version` conditional.

Reply:
Done. Use `%{_datadir}` instead of `/usr/share`.